src/connectors/autocomplete/__tests__/connectAutocomplete-test.ts

```diff
@@ -429,14 +429,14 @@ search.addWidgets([
     });
   });
 
-  describe('getWidgetRenderState', () => {
+  describe('getRenderState', () => {
     test('returns the render state', () => {
       const renderFn = jest.fn();
       const unmountFn = jest.fn();
       const createAutocomplete = connectAutocomplete(renderFn, unmountFn);
       const autocomplete = createAutocomplete({});
 
-      const renderState1 = autocomplete.getWidgetRenderState!(
+      const renderState1 = autocomplete.getRenderState!(
```

Comment:
also in the tests it can be removed

Reply:
Hmm. To remove the exclamation mark from `getRenderState`, we should go like this:

![image](https://user-images.githubusercontent.com/499898/95346016-a8460600-08bb-11eb-8a7d-6a28e29ff3f3.png)


What do you think?

Reply:
yes, I think that makes sense